src/Logging/HTTPOutputHandler.php

```diff
@@ -70,6 +72,38 @@ public function setStatusCode($statusCode)
     }
 
     /**
+     * Set a formatter to use if Director::is_cli() is true
```

Comment:
I think @sminnee may be right... maybe I've been doing it wrong all these years? :P